src/pages/public_registration/ActionPanelContent.tsx

```diff
@@ -10,39 +10,44 @@ import { ErrorBoundary } from '../../components/ErrorBoundary';
 import { setNotification } from '../../redux/notificationSlice';
 import { RootState } from '../../redux/store';
 import { PublishingTicket, Ticket } from '../../types/publication_types/ticket.types';
-import { RegistrationStatus } from '../../types/registration.types';
 import { isErrorStatus, isSuccessStatus } from '../../utils/constants';
-import { getTitleString, userHasAccessRight } from '../../utils/registration-helpers';
+import { getTitleString } from '../../utils/registration-helpers';
 import { UrlPathTemplate } from '../../utils/urlPaths';
 import { PublicRegistrationContentProps } from './PublicRegistrationContent';
 import { DoiRequestAccordion } from './action_accordions/DoiRequestAccordion';
 import { PublishingAccordion } from './action_accordions/PublishingAccordion';
 import { SupportAccordion } from './action_accordions/SupportAccordion';
 
 interface ActionPanelContentProps extends PublicRegistrationContentProps {
-  tickets: Ticket[];
   refetchData: () => Promise<void>;
   isLoadingData?: boolean;
+  shouldSeePublishingAccordion: boolean;
+  shouldSeeDoiAccordion: boolean;
+  shouldSeeSupportAccordion: boolean;
+  shouldSeeDelete: boolean;
+  publishingRequestTickets: PublishingTicket[];
+  newestDoiRequestTicket?: Ticket;
+  newestSupportTicket?: Ticket;
 }
```

Comment:
dessverre litt for mange props her, men ser ikke noen bedre løsning. :/ 
Dumt å kalkulere alle disse enda en gang, siden de også trengs i parent, og virker overkill å legge det i en ny react context. Så da ble det en litt skitten løsning her nå, om ingen har bedre forslag